snapcraft/models/project.py

```diff
@@ -571,14 +571,14 @@ def _validate_grade_and_build_base(cls, values):
             raise ValueError("grade must be 'devel' when build-base is 'devel'")
         return values
 
-    @pydantic.validator("base", always=True)
+    @pydantic.root_validator()
```

Comment:
we should pull this fix into the hotfix branch

Reply:
yes